Added Author Feature and additional description to Contrib section of READ.md

McCune1224 opened the pull request:
Features added as mentioned in #6. LMK if you want anything tweaked. Tried to maintain the same syntax used within the project when adding HTML and CSS associated with the author.

N-Shar-ma commented on styles.css:
```diff
@@ -119,6 +119,13 @@ main{
 }
 
+#deck-author-display {
+    position: absolute;
```
Setting the position again is redundant, so could you please remove this line?

N-Shar-ma reviewed (changes requested):
@McCune1224 The code and result look great! Especially appreciate the helpful boilerplate added to the readme :)

It seems you (probably unintentionally) linted all the files due to which there are a lot of white space / tab changes. It's not too big a deal but if you could undo those, it would be nice, so the diff only has the relevant code changes.

One further improvement that just struck me is that it would be nice for the author name to be a link to their GitHub. So instead of a span, could you make it an `a` element with `href` set to the author's GitHub profile? It should look the same as it does now, just an underline on hover to indicate it's a link would be cool.

Sidenote: Adding screenshots showing visual changes is always nice, so please add that when you've made the author's name a link! A screenshot each for non hover and hover state would be nice

McCune1224 commented:
Thanks for the feedback, I'll tweak my editor to do 2 tab instead of 4 tab. I'll also go ahead and change the author name to contain a link to their github.

McCune1224 commented:
Alrighty, made the changes requested:

1.Javascript/HTML:
-  Changed Span to A tag with href to author's github page and updated JS Dom manipulation
-  Fixed 4 tab linting issue, Javascript code should be back to 2 tab now. 
2.CSS: 
-  Added Hover effect for link in css to darken on hover and removed

McCune1224 commented:
# For the CSS on hover the text will be dimmed:

#Non-Hovered
![Deck Created non-hover](https://user-images.githubusercontent.com/19767251/193479408-79eb35e1-8b64-43a4-a514-a74263df0c91.png)

# Hovered (link displayed in bottom left)
![Deck Created hover](https://user-images.githubusercontent.com/19767251/193479414-b8815a8a-3353-4651-8ad5-a12a7bd2b70e.png)

N-Shar-ma commented:
@McCune1224 Looks like you may have forgotten to push? 😅 

Seeing the screenshots, could you make the link have an underline on hover instead of darkening it? Also please make it the same red like the rest of the text and not the default blue.

And in case you haven't already please remove the redundant `position: absolute` from the CSS

McCune1224 commented:
Oops, yeah that would help to push it... Anywho made the additional CSS request to maintain the red and only underline on hover, here's what it looks like now

# non-hover 
![Deck Created non-hover](https://user-images.githubusercontent.com/19767251/193480601-d3824b0b-da33-4d1b-b916-013801415b4b.png)

# hover
![Deck Created hover](https://user-images.githubusercontent.com/19767251/193480596-eb36b9c1-f900-4d46-a3b5-7f63c5374ef2.png)

N-Shar-ma commented:
Merged after a couple minor fixes. Thanks a lot @McCune1224 ! 🚀
